chainerx_cc/chainerx/cuda/cuda_device/linalg.cu

```diff
@@ -213,14 +213,15 @@ void SolveImpl(const Array& a, const Array& b, const Array& out) {
     auto ipiv_ptr = static_cast<int*>(internal::GetRawOffsetData(ipiv));
 
     int buffersize = 0;
-    device_internals.cusolverdn_handle().Call(GetrfBuffersize<T>, m, m, lu_ptr, m, &buffersize);
+    device_internals.cusolverdn_handle().Call(GetrfBuffersize<T>, m, m, lu_ptr, std::max(int64_t{1}, m), &buffersize);
```

Comment:
Not a strong opinion but how about assigning `std::max(int64_t{1}, m)` to a local variable?